finesse/hardware/plugins/time/__init__.py

```diff
@@ -1 +1,23 @@
 """This module contains interfaces for querying time sources."""
+
+from datetime import datetime
+
+from finesse.hardware.manage_devices import get_device_instance
+from finesse.hardware.plugins.time.time_base import TimeBase
+
+
+def get_time_instance() -> TimeBase | None:
+    """Get the instance of the current time source if connected or None."""
+    return get_device_instance(TimeBase)
+
+
+def get_time() -> datetime:
```

Comment:
This is a bit pedantic, but maybe this function should be called `get_current_time()`? Otherwise the meaning may not always be clear from context.